v2/go.mod

```diff
@@ -10,5 +10,6 @@ require (
 
 require (
 	github.com/go-logr/logr v0.2.0 // indirect
+	github.com/google/go-cmp v0.6.0 // indirect
```

Comment:
This change is optional but I figured it'd be more useful to use `cmp.Diff` instead of a two complex structs that are quite hard to read in the tests. k/k already has this dependency https://github.com/kubernetes/kubernetes/blob/master/go.mod#L38 so we are not adding anything new for them.